Meadowlark-js/packages/metaed-plugin-edfi-meadowlark/src/enhancer/JoiSchemaEnhancer.ts

```diff
@@ -228,13 +236,20 @@ function buildJoiSchema(entityForSchema: TopLevelEntity): Joi.AnySchema {
  * shape for each resource that corresponds to the MetaEd entity.
  */
 export function enhance(metaEd: MetaEdEnvironment): EnhancerResult {
+  // Build schemas for each domain entity and association
   getAllEntitiesOfType(metaEd, 'domainEntity', 'association', 'domainEntitySubclass', 'associationSubclass').forEach(
     (entity) => {
       const entityMeadowlarkData = entity.data.meadowlark as EntityMeadowlarkData;
       entityMeadowlarkData.joiSchema = buildJoiSchema(entity as TopLevelEntity);
     },
   );
 
+  // Attach descriptor schema to each descriptor
+  getAllEntitiesOfType(metaEd, 'descriptor').forEach((entity) => {
+    const entityMeadowlarkData = entity.data.meadowlark as EntityMeadowlarkData;
+    entityMeadowlarkData.joiSchema = descriptorSchema;
+  });
+
```

Comment:
Descriptors all have the same schema

Reply:
As an aside, everywhere you see `'descriptor'` in this PR, it's a Typescript string literal type that enumerates the valid MetaEd types.